internal/impl/redis/output_streams.go

```diff
@@ -90,6 +92,9 @@ func newRedisStreamsWriter(conf output.RedisStreamsConfig, log log.Modular) (*re
 	if r.metaFilter, err = conf.Metadata.Filter(); err != nil {
 		return nil, fmt.Errorf("failed to construct metadata filter: %w", err)
 	}
+	if r.interpolatedStreamName, err = mgr.BloblEnvironment().NewField(conf.Stream); err != nil {
```

Comment:
Maybe move this above the `metaFilter` part, so they're in the same order as the fields in the struct

Reply:
Moved :)